hack/gen-tf-code/main.go

```diff
@@ -358,6 +358,26 @@ func main() {
 		),
 		generate(kops.InstanceRootVolumeSpec{}),
 		generate(kops.OIDCAuthenticationSpec{}),
+		// 1.31
+		generate(kops.ClusterField{}),
+		generate(corev1.EnvVar{},
+			required("Name"),
+		),
+		generate(corev1.EnvVarSource{}),
+		generate(corev1.ObjectFieldSelector{},
+			required("FieldPath"),
+		),
+		generate(corev1.ResourceFieldSelector{},
+			required("Resource"),
+			exclude("Divisor"),
```

Comment:
Not really happy about this one but it gave strange errors without it. The resource.Quantity-type used for the Divisor is strange and has no exported fields.

Reply:
Hey @argoyle from what i understand `resource.Quantity` should serialise as `string`.

Reply:
What was the issue when `Divisor` is not excluded ?

Reply:
It's an error from go vet on a generated test:
```
vet: pkg/schemas/core/Resource_ResourceFieldSelector.generated_test.go:90:24: cannot use "" (untyped string constant) as resource.Quantity value in assignment
```